src/ncp/ncp_base.cpp

```diff
@@ -41,14 +41,25 @@
 #include <platform/radio.h>
 #include <platform/misc.h>
 
-namespace Thread {
+namespace Thread
+{
 
 extern ThreadNetif *sThreadNetif;
 
 static NcpBase *sNcpContext = NULL;
 
 #define NCP_PLAT_RESET_REASON        (1<<31)
 
+enum
+{
+    kThreadModeTLV_Receiver    = (1 << 3),
+    kThreadModeTLV_Secure      = (1 << 2),
+    kThreadModeTLV_DeviceType  = (1 << 1),
+    kThreadModeTLV_NetworkData = (1 << 0),
+};
+
+#define UNDEFINED_RSSI         127 // Used for PROP_MAC_WHITELIST
```

Comment:
I will update `PROTOCOL.md`.

But just to clarify, you can add things to the whitelist either with or without the RSSI-override value being present. If it isn't present, or if it is present but the value is 127, it is taken as an indication that the `otAddMacWhitelist()` API should be used instead of the `otAddMacWhitelistRssi()` API.

When fetching the list of all whitelist entries, the RSSI-override is either set to the actual RSSI-override value, or it is set to 127 to indicate that RSSI-override is disabled for this address.

Make sense?